mu4/notation/view/resources/data/std_sample.mscx

```diff
@@ -0,0 +1,192 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Where do we use this file?

Reply:
What is the purpose?

Reply:
This file(and tab_sample.mscx) is needed to view changes in notation in real-time on test data, not on real data.
A new score is generated from this file and loaded to ExampleView.